ruby/vampires.rb

```diff
@@ -0,0 +1,56 @@
+puts "How many employees you will be processed?"
+num_empl = gets.chomp.to_i 
+
+
+i=0
+until i == num_empl
+
+	puts "What is your name?"
+	name = gets.chomp
+	name == "Drake Cula" || name == "Tu Fang"? name = true : name = false
+	
+	puts "What is your age? What year were you born?"
+	age = gets.chomp.split(' ').map(&:to_i)
```

Comment:
What type of user input are you getting here? If you are expecting complex user input you should consider having an example.